app/controllers/content_pages_controller.rb

```diff
@@ -18,7 +18,7 @@ def show
     elsif module_item.assessment_results?
       redirect_to training_module_assessment_result_path(training_module_name, module_item)
     else
-      render content_page_partial(module_item)
+      render_content_page_partial
     end
```

Comment:
Technically rendering a template, not a partial - can just name this `render_content_page` or `render_page`

Reply:
Quite right good idea